dashboard/test/controllers/rubrics_controller_test.rb

```diff
@@ -66,4 +66,43 @@ class RubricsControllerTest < ActionController::TestCase
     refute_nil LearningGoalEvaluation.find_by(user: student, teacher: teacher).submitted_at
     assert_nil LearningGoalEvaluation.find_by(user: student, teacher: another_teacher).submitted_at
   end
+
+  test "gets ai evaluations for student and learning goal" do
+    student = create :student
+    teacher = create :teacher
+    create :follower, student_user: student, user: teacher
+    sign_in teacher
+
+    rubric = create :rubric
+    learning_goal1 = create :learning_goal, rubric: rubric
+    learning_goal2 = create :learning_goal, rubric: rubric
+    ai_evaluation1 = create :learning_goal_ai_evaluation, learning_goal: learning_goal1, user: student, understanding: 1
+    ai_evaluation2 = create :learning_goal_ai_evaluation, learning_goal: learning_goal2, user: student, understanding: 2
```

Comment:
not blocking for this PR, but we should probably add constants to refer to these understanding values. looks like we actually have some already, thoughts on whether to reuse these? https://github.com/code-dot-org/code-dot-org/blob/472deb8c693c54512379775c6c9a4e09a99952dc/lib/cdo/shared_constants.rb#L601-L608

Reply:
oh yep! Was planning on using those but was planning on doing so in a follow-up PR if that's alright?